### ISSUE TITLE
npm installing results in an UNMET PEER DEPENDENCY error

### ISSUE THREAD

**Issue by stringbeans** [kuhnza/angular-google-places-autocomplete#78]
npm installing results in an `UNMET PEER DEPENDENCY grunt@~0.4.1` error:

```
├─┬ angular-google-places-autocomplete@0.2.7  (git+ssh://git@github.com/kuhnza/angular-google-places-autocomplete.git#61cf02e7db1e5c7f01dace0ba58ce8912066aa12)
│ ├── UNMET PEER DEPENDENCY grunt@~0.4.1
│ └─┬ grunt-contrib-cssmin@0.10.0 
│   ├─┬ clean-css@2.2.23 
│   │ └── commander@2.2.0 
│   └─┬ maxmin@0.2.2 
│     ├─┬ chalk@0.5.1 
│     │ ├── ansi-styles@1.1.0 
│     │ ├─┬ has-ansi@0.1.0 
│     │ │ └── ansi-regex@0.2.1 
│     │ ├── strip-ansi@0.3.0 
│     │ └── supports-color@0.2.0 
│     ├── gzip-size@0.2.0 
│     └── pretty-bytes@0.1.2 
```

**Comment by laurence-myers** [comment]
Any idea how to resolve this?

**Comment by laurence-myers** [comment]
For posterity, here's how we resolved it:

The NPM package is actually [a fork, from here](https://github.com/twaite/angular-google-places-autocomplete), by user twaite.

The fix requires altering package.json and moving "grunt-contrib-cssmin" to a dev dependency.

[Here's a commit showing how we fixed it.](https://github.com/ltnetwork/angular-google-places-autocomplete/commit/f74c3386a74c56fa6e54df42d004ccb3ffab64ee)

**Comment by IgorKurkov** [comment]
@laurence-myers it would be great if you make just another one package with this fix :) like ltn-angular-g-p-auto ....

**Comment by laurence-myers** [comment]
@IgorKurkov Unfortunately, I no longer work on the application that used this library, and I don't want to become the maintainer for something I don't use. 😞

**Comment by IgorKurkov** [comment]
yeah sure! :) Old applications cause a lot of problems. Thank you for your fix, I already forked it for freezing this case by using `npm i <username>/<githubrepo>#<branch>` directly
